src/styles/sub-nav.css

```diff
@@ -0,0 +1,19 @@
+.sub-nav {
+  width: 100%;
+  position: -webkit-sticky;
+  position: sticky;
+  top: 0;
+  z-index: 10;
+  display: flex;
+  justify-content: space-between;
+  padding: 1rem 0.5rem; }
+  .sub-nav #micasa-logo {
+    width: 75%; }
+  .sub-nav img {
+    width: 100%; }
+  .sub-nav .buy-sell p:first-child, .sub-nav .buy-sell p:nth-child(2) {
+    border-right: 1px solid white;
+    padding-right: 10px; }
+  .sub-nav .buy-sell p {
```

Comment:
can you add a `:hover` state to change the cursor to a pointer when hovering over a link?